### PR TITLE
Db connector fixes

### LINKED ISSUE #131: Many rails tables have empty columns
If you navigate to a property show page in the web app and QA the user interface by clicking to view the tables, you will notice there are many blank columns.  When PR #130 is merged, you will have access to the table endpoints to look at this, but for now the quickest way to see this is to run the app at `localhost:3000` and view a property.  You could try 1749 Grand Concourse, Bronx for starters.

Many tables have blank columns, but for example, here is DOB Permits:
image

### PR DESCRIPTION
**Pull request by rdlester**
Fixes #131 (????)

There are still a few empty columns in r_hpd_complaints; filling these in is a larger task involving joining the complaints and complaints_problems tables. I've opened a separate issue for that.

### CODE REVIEW

File: angular-rails-app/db/schema.rb [hunk 1]
```diff
@@ -43,7 +43,7 @@
     t.datetime "job_start_date"
     t.string   "job_type",        limit: 255
     t.string   "job_num",         limit: 255
-    t.string   "filling_status",  limit: 255
+    t.string   "filing_status",  limit: 255
```
Comment (chrishenry) [inline angular-rails-app/db/schema.rb:46]: So that's why that looked so weird. 😉

File: angular-rails-app/lib/tasks/db_connector.rake [hunk 2]
```diff
@@ -322,25 +324,25 @@ namespace :db_connector do
...
+      "LL6291", # -LOCAL LAW 62/91 - BOILERS",
+      "C", # -CONSTRUCTION",
+      "B", # -BOILER",
     ]
 
     in_list =  "\"" + v_types.join("\", \"") + "\""
 
     puts "Pulling dob violations..."
-    sql = "SELECT * FROM dob_violations WHERE violation_type IN (#{in_list})"
+    sql = "SELECT * FROM dob_violations WHERE violation_type_code IN (#{in_list})"
```
Comment (chrishenry) [inline angular-rails-app/lib/tasks/db_connector.rake:345]: Nice, this'll probably perform better. You should change the `add_indexes` params above to match this column.

**Review by chrishenry — CHANGES_REQUESTED** [review]

File: angular-rails-app/lib/tasks/db_connector.rake (hunk 2 above)
Comment (rdlester) [inline angular-rails-app/lib/tasks/db_connector.rake]: Done.

Comment (rdlester) [reply angular-rails-app/lib/tasks/db_connector.rake:345]: Done.

Comment (chrishenry) [reply angular-rails-app/lib/tasks/db_connector.rake:345]: sweet. 👍

**Review by chrishenry — APPROVED** [review]

**Pull request merged**